tests/unit_tests/commands_test.coffee

```diff
@@ -1,5 +1,6 @@
 require "./test_helper.js"
 extend global, require "./test_chrome_stubs.js"
+global.Settings = {postUpdateHooks: {}}
```

Comment:
I know there's a lot of legacy usage around, but does the style guide not say...

```Coffeescript
global.Settings = { postUpdateHooks: {} }
```

(That is, with spaces.  I usually try to tidy these up en passant.)